src/Microsoft.ML.AutoML/AutoMLExperiment/IDatasetManager.cs

```diff
@@ -4,18 +4,18 @@
 
 namespace Microsoft.ML.AutoML
 {
-    internal interface IDatasetSettings
+    public interface IDatasetManager
```

Comment:
For the public interfaces can you add just a very brief description of what they are for?